.gitignore

```diff
@@ -11,3 +11,4 @@ lerna-debug.log
 sassdoc-data.json
 backpack.iml
 yarn.lock
+/packages/*/changelog.md
```

Comment:
Is this just temporary? My understanding is that we'll still have `changelod.md`'s in the repo to link to etc?

Reply:
Yes this is just temporary, let's get rid of this later